.travis.yml

```diff
@@ -11,3 +11,7 @@ branches:
   only:
     - master
     - 1.x_dev
+
+script:
+  - xcodebuild -workspace push-sdk.xcworkspace -scheme push-sdk -sdk iphonesimulator build
+  - xcodebuild -workspace push-sdk.xcworkspace -scheme push-sdk -destination 'platform=iOS Simulator,name=iPhone 7,OS=10.0' test
```

Comment:
Nothing related to iOS 9. This is going to run the tests on an iPhone 7 using iOS 10.0 (OS system)

I have added it because we didn't have any tested before :D